File: benchmarks/suite5/alu.go
package main


func main() {
	n := suite5OpaqueI64(2000000)
	var acc int64
	var i int64
	for i = 0; i < n; i++ {
		acc = acc + i*3 - i/2 + i%7
	}
	suite5PrintI64(acc)
}

File: benchmarks/suite5/alu.rs
mod bench_runtime;
use bench_runtime::suite5_print_i64;

fn main() {
    let n: i64 = 2_000_000;
    let mut acc: i64 = 0;
    let mut i: i64 = 0;
    while i < n {
        acc = acc + i * 3 - i / 2 + i % 7;
        i += 1;
    }
    suite5_print_i64(acc);
}
